src/decays/decay.hpp

```diff
@@ -286,6 +286,9 @@ final_state_symmetry_factor(typename cxx_diagrams::field_indices<Field1>::type c
    }
 }
 
+struct my_f_params {double mHOS; double mVOS; double GammaV;};
```

Comment:
Personally I'd prefer to default-initialize all members of such structs, for example as:
~~~.cpp
struct my_f_params {
  double mHOS{};
  double mVOS{};
  double GammaV{};
};
~~~
This avoids bugs from uninitialized values.

Reply:
You're right. Also, the name was awful. It was ok when it was a local type, but now it's not. I've renamed it to `hVV_4body_params`.

Reply:
Thanks!